ballerina/api.bal

```diff
@@ -17,6 +17,9 @@
 import ballerina/jballerina.java;
 
 configurable DefaultModelConfig? defaultModelConfig = ();
+public type Schema record {
+    json schema;
```

Comment:
Isn't schema always a JSON object?

Reply:
Yes. It is a JSON object. I defined it as a record so that we can easily add new fields if we need additional information in the future. If that's not necessary, I can change it to JSON

Reply:
We can use `map<json>` for JSON object.

Reply:
In future we may have metadata to store, like version and those details

Reply:
Updated to map<json>